features/mbedtls/importer/adjust-check-config.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+#
+# This file is part of mbed TLS (https://tls.mbed.org)
+#
+# Copyright (c) 2019, Arm Limited, All Rights Reserved
+#
+# Purpose
+#
+# Removes checks from check_config.h that aren't needed for Mbed OS
+#
+# Usage: adjust-check-config.sh [path to check_config file]
+#
+set -eu
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 path/to/check_config.h" >&2
+    exit 1
+fi
+
+FILE=$1
+
+conf() {
+    $SCRIPT -f $FILE --force $@
+}
+
+remove_code() {
+    MATCH_PATTERN=$(IFS=""; printf "%s" "$*")
+
+    perl -0pi -e "s/$MATCH_PATTERN//g" "$FILE"
+}
+
+remove_code                                                                                 \
```

Comment:
Please add a comment explaining why this is removed. It'd be good for each modification done by this script to have the rationale, so that we can understand and are able to change it in the future.